01.fizzbuzz/fizz_buzz.rb

```diff
@@ -0,0 +1,12 @@
+
+for i in 1..20 do
```

Comment:
Rubyでは以下のように書くことも可能で、こちらの方が推奨されている感があるので覚えておくといいかもしれません。
```suggestion
(1..20).each do |i|
```